bodhi/server/models/models.py

```diff
@@ -1260,11 +1264,26 @@ def modify_bugs(self):
                     # https://github.com/fedora-infra/bodhi/issues/368#issuecomment-135155215
                     for bug in self.bugs:
                         if not bug.parent:
-                            log.debug("Closing tracker bug %d" % bug.bug_id)
-                            bug.close_bug(self)
+                            bad, good = self.get_bug_karma(bug)
+                            bug_karma = good + bad
+                            if bug_karma >= 0:
+                                log.debug("Closing tracker bug %d" % bug.bug_id)
+                                bug.close_bug(self)
+                            else:
+                                log.debug('Adding stable comment to bugs for %s', self.title)
+                                bug.add_comment(self)
```

Comment:
@anassahmed yeah that does sound like a lot of work - don't worry about it. Maybe I'll do it later in a follow on commit ☺